plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/DeltaLakeSchemaSupport.java

```diff
@@ -174,6 +195,12 @@ public static ColumnMappingMode getColumnMappingMode(MetadataEntry metadata, Pro
         return Enums.getIfPresent(ColumnMappingMode.class, columnMappingMode.toUpperCase(ENGLISH)).or(ColumnMappingMode.UNKNOWN);
     }
 
+    public static IsolationLevel getIsolationLevel(MetadataEntry metadata)
+    {
+        String isolationLevel = metadata.getConfiguration().getOrDefault(ISOLATION_LEVEL_CONFIGURATION_KEY, IsolationLevel.WRITESERIALIZABLE.getValue());
+        return Enums.getIfPresent(IsolationLevel.class, isolationLevel.toUpperCase(ENGLISH)).or(IsolationLevel.UNKNOWN);
```

Comment:
"unknown" will be written in `isolationLevel` field if  `delta.isolationLevel` has unsupported value. 

We can just throw an exception for unsupported value as this method is called only from `finishInsert` method.

Reply:
Also, can we leave a code comment why WriteSerializable is the default?